orm/encoding/ormfield/bytes.go

```diff
@@ -58,7 +52,13 @@ func (b NonTerminalBytesCodec) FixedBufferSize() int {
 
 func (b NonTerminalBytesCodec) ComputeBufferSize(value protoreflect.Value) (int, error) {
```

Comment:
let's add a doc:
_ComputeBufferSize returns the bytes size of a serialized value._

Reply:
there is a doc string on the interface method. do we need it on each implementation?

Reply:
Usually we should point to the interface name. eg:
```
ComputeBufferSize implements InterfaceName
```